crates/precompiles/src/tip20/mod.rs

```diff
@@ -899,26 +805,23 @@ impl TIP20Token {
 
         self.check_spending_limit(from, amount)?;
 
-        // Handle rewards (only after Moderato hardfork)
-        if self.storage.spec().is_moderato() {
-            // Accrue rewards up to current timestamp
-            let current_timestamp = self.storage.timestamp();
-            self.accrue(current_timestamp)?;
-
-            // Update rewards for the sender and get their reward recipient
-            let from_reward_recipient = self.update_rewards(from)?;
-
-            // If user is opted into rewards, decrease opted-in supply
-            if from_reward_recipient != Address::ZERO {
-                let opted_in_supply = U256::from(self.get_opted_in_supply()?)
-                    .checked_sub(amount)
-                    .ok_or(TempoPrecompileError::under_overflow())?;
-                self.set_opted_in_supply(
-                    opted_in_supply
-                        .try_into()
-                        .map_err(|_| TempoPrecompileError::under_overflow())?,
-                )?;
-            }
+        // Accrue rewards up to current timestamp
+        let current_timestamp = self.storage.timestamp();
+        self.accrue(current_timestamp)?;
```

Comment:
Noting that this should be removed as we remove all storage variables related to streaming rewards. We can do this in a subsequent PR.